app/assets/javascripts/admin/addon/components/theme-translation.js

```diff
@@ -1,16 +1,23 @@
 import SiteSettingComponent from "./site-setting";
+import { ajax } from "discourse/lib/ajax";
+import { url } from "discourse/lib/computed";
 import { alias } from "@ember/object/computed";
 
-export default class ThemeTranslation extends SiteSettingComponent {
+export default class extends SiteSettingComponent {
```

Comment:
Was this an intentional change?

```suggestion
export default class ThemeTranslation extends SiteSettingComponent {
```

Reply:
Good catch, this was an oversight on my part.